### PR TITLE
2020 12 msa feature

### PR DESCRIPTION
**Pull request by dlrice**
- [x] What is the link to the Jira that this PR is for?
[Clicking on a feature in the protvista-tracks in BLAST and Align should trigger the display the corresponding tooltip.](https://www.ebi.ac.uk/panda/jira/browse/TRM-24863)
- [x] Have you written tests and do these have >= 75% coverage of the code that you are contributing?
- [x] Is it possible to visually evaluate this PR in the browser? If so, what are the steps to do this (eg specific URLs or sections)?
Both align and BLAST results
- [x] Are there any linting errors that required you to add exceptions?
Do not consider `useCustomElement`'s argument (which is just an import function) for the code coverage calculations.

# Change Summary
* Clicking on a feature in either track or MSA opens `protvista-tooltip`.
* Use `protvista-feature-adapter`'s `formatTooltip` to try to recreate content display of existing protvista's tooltip. Data passed to `prepareFeatureForTooltip` before passing `formatTooltip`.
* MSA feature border color reflects track feature color (`getColorByType` now exported from 'protvista-track').
* Feature track label includes active accession.

### CODE REVIEW

**Review by aurel-l — CHANGES_REQUESTED** [review]
Globally OK with the majority of changes.

Some small comments here and there.

And I found one edge case where if none of the sequences has annotations (edited sequences, or accession-less sequences). In the wrapped view, the spaces allocated to the feature tracks in the grid is occupied by the alignment tracks and it's all messed up.
It should maybe render an empty element to use this space in the grid, instead of rendering undefined.

File: src/tools/components/Wrapped.tsx [hunk 1]
```diff
@@ -79,82 +87,93 @@ const WrappedRow: FC<WrappedRowProps> = ({
...
+            )
+          )
+          .filter(Boolean);
       }
     },
     // TODO: replace this with fragments to have one big grid
     // -> to keep the right column of the right size to fit all possible values
-    [trackDefined, annotation, activeSeq]
+    [
+      activeAlignment?.from,
```
[the remark this answers is not in the document]
Comment (dlrice) [reply src/tools/components/Wrapped.tsx]: The `activeAlignment?.*` was auto-suggested and we've changed this to now consider `activeAlignment` instead.

**Comment by dlrice** [comment]
Addressed all comments. Ready for re-review.

File: src/tools/components/AlignmentView.tsx [hunk 2]
```diff
@@ -197,6 +263,111 @@ const AlignmentView: React.FC<{
...
+    const className = '.main-content-and-footer';
+    const mainContentAndFooter = document.querySelector(className);
+    if (!mainContentAndFooter) {
+      throw Error(`Cannot find :${className}`);
+    }
+    const handleScroll = () => {
+      setTooltipContent(null);
+    };
+    mainContentAndFooter.addEventListener('scroll', handleScroll, {
+      passive: false,
```
[the remark this answers is not in the document]
Comment (dlrice) [reply src/tools/components/AlignmentView.tsx]: Needed for when the page is scrollable and the user scrolls which will leave the tooltip fixed in place.

**Review by aurel-l — APPROVED** [review]

**Pull request merged**